pkg/synthetictests/duplicated_events_special.go

```diff
@@ -91,7 +91,7 @@ func newSingleEventCheckRegex(testName, regex string, failThreshold, flakeThresh
 //
 // to happen over a certain threshold and marks it as a failure or flake accordingly.
 func testBackoffPullingRegistryRedhatImage(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
-	testName := "[sig-arch] should not see excessive pull back-off on registry.redhat.io"
+	testName := "[sig-arch] pathological event should not see excessive pull back-off on registry.redhat.io"
```

Comment:
If you are renaming tests, you need to decide if the rename is important, and then have a companion PR to ci-test-mapping to ensure we account for them all. Example PR from David https://github.com/openshift-eng/ci-test-mapping/pull/34

Reply:
I'd also suggest making the test names an understandable phrase.  In this case, the pathological event is not what is seeing excessive pull back-off.  That applies to the rest of the renames. Maybe do something like `[PathologicalEvent]`?

Reply:
https://github.com/openshift-eng/ci-test-mapping/pull/38